### PR TITLE
(DOCS-2753) Remove instances of 'easy' and 'easily'

### PR DESCRIPTION
**Pull request by jtappa**
### What does this PR do?
Where possible, this removes all instances of "easy" and "easily" from the docs. We don't want to assume anything is easy for our users or readers. Places where these terms might be acceptable are when you're making a direct comparison, which is why all instances of "easier" were left, and some instances of "easy" or "easier" were modified to use "easier" instead.

### Motivation

OKR

### Preview

https://docs-staging.datadoghq.com/<BRANCH_NAME>/<PATH>

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by jtappa** [comment]
Review request from serverless-product was auto-added, likely not necessary in this case.

File: content/en/monitors/faq/i-have-a-downtime-scheduled-on-my-monitor-why-did-it-still-alert.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ When you [schedule a downtime][1] over a specific tag scope, the downtime will a
 
 If you want to schedule a downtime over a subset of hosts, a good way to do so is to find a host-tag common to all those hosts and scope the downtime by that tag.
 
-One easy way you can take advantage of this is to create new host-tags directly in the Datadog UI from your [Infrastructure List][2] and [Host Map][3] by selecting a host and hitting the **Edit Tags** button (shown below), or via [our API][4].
+One way you can take advantage of this is to create new host-tags directly in the Datadog UI from your [Infrastructure List][2] and [Host Map][3] by selecting a host and hitting the **Edit Tags** button (shown below), or via [our API][4].
```
Comment (ruthnaebeck) [inline content/en/monitors/faq/i-have-a-downtime-scheduled-on-my-monitor-why-did-it-still-alert.md:20]:
```suggestion
One way you can take advantage of this is to create new host-tags directly in the Datadog UI from your [Infrastructure List][2] and [Host Map][3] by selecting a host and hitting the **Edit Tags** button (shown below), or with the [Datadog API][4].
```

File: content/en/real_user_monitoring/guide/send-rum-custom-actions.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ All RUM context will be automatically attached (current page view information, g
 ## Create facets and measures on your new attributes
 Once you have deployed the code that creates your custom actions, you will start seeing actions appear in the [RUM Explorer][3], in the **Actions** tab.
 
-To easily filter on your new custom Actions, use the `Action Target Name` attribute as follow: `@action.target.name:<ACTION_NAME>`. In the example, we use the following filter: `@action.target.name:checkout`
+To filter on your new custom Actions, use the `Action Target Name` attribute as follow: `@action.target.name:<ACTION_NAME>`. In the example, we use the following filter: `@action.target.name:checkout`
```
Comment (ruthnaebeck) [inline content/en/real_user_monitoring/guide/send-rum-custom-actions.md:32]:
```suggestion
To filter on your new custom Actions, use the `Action Target Name` attribute as follow: `@action.target.name:<ACTION_NAME>`. The example uses the following filter: `@action.target.name:checkout`
```

File: content/en/serverless/guide/extension_motivation.md [hunk 3]
```diff
@@ -33,7 +33,7 @@ The Lambda extension has full feature parity with the Forwarder for Python, Node
 
 The Datadog Lambda Extension offers the following advantages over the Datadog Forwarder:
 
-- **Simple to set up**: The Forwarder requires that triggers need be set up for every new Lambda function. The Datadog Lambda extension can be easily added as a Lambda layer. Unlike with the Forwarder, configuring the Lambda extension does not require permissions to install third-party AWS CloudFormation stacks. Also, the Lambda extension sends telemetry directly to Datadog, so you do not need to manage CloudWatch Log group subscriptions for your Lambda functions.
+- **Simple to set up**: The Forwarder requires that triggers need be set up for every new Lambda function. The Datadog Lambda extension can be added as a Lambda layer. Unlike with the Forwarder, configuring the Lambda extension does not require permissions to install third-party AWS CloudFormation stacks. Also, the Lambda extension sends telemetry directly to Datadog, so you do not need to manage CloudWatch Log group subscriptions for your Lambda functions.
```
Comment (ruthnaebeck) [inline content/en/serverless/guide/extension_motivation.md:36]:
```suggestion
- **Simple to set up**: The Forwarder requires triggers to be set up for every new Lambda function. The Datadog Lambda extension can be added as a Lambda layer. Unlike with the Forwarder, configuring the Lambda extension does not require permissions to install third-party AWS CloudFormation stacks. Also, the Lambda extension sends telemetry directly to Datadog, so you do not need to manage CloudWatch Log group subscriptions for your Lambda functions.
```

File: content/en/tracing/guide/add_span_md_and_graph_it.md [hunk 4]
```diff
@@ -231,7 +231,7 @@ The Datadog UI uses tags to set span level metadata. Custom tags may be set for
 
 {{< img src="tracing/guide/add_span_md_and_graph_it/span_md_3.png" alt="Resource Page"  style="width:90%;">}}
 
-The Trace table shows you both the overall latency distribution of all traces in the current scope (service, resource and timeframe) and links to individual traces. You can sort this table by duration or error code to easily identify erroneous operation or opportunities for optimization.
+The Trace table shows you both the overall latency distribution of all traces in the current scope (service, resource and timeframe) and links to individual traces. You can sort this table by duration or error code to identify erroneous operation or opportunities for optimization.
```
Comment (ruthnaebeck) [inline content/en/tracing/guide/add_span_md_and_graph_it.md:234]:
```suggestion
The Trace table shows you both the overall latency distribution of all traces in the current scope (service, resource, and timeframe) and links to individual traces. You can sort this table by duration or error code to identify erroneous operation or opportunities for optimization.
```

File: content/en/tracing/setup_overview/custom_instrumentation/ruby.md [hunk 5]
```diff
@@ -165,7 +165,7 @@ end
 
 ## Adding spans
 
-If you aren't using supported library instrumentation (see [library compatibility][4]), you may want to to manually instrument your code. Adding tracing to your code is easy using the `Datadog.tracer.trace` method, which you can wrap around any Ruby code:
+If you aren't using supported library instrumentation (see [library compatibility][4]), you may want to to manually instrument your code. Add tracing to your code by using the `Datadog.tracer.trace` method, which you can wrap around any Ruby code:
```
Comment (ruthnaebeck) [inline content/en/tracing/setup_overview/custom_instrumentation/ruby.md:168]:
```suggestion
If you aren't using supported library instrumentation (see [library compatibility][4]), you can manually instrument your code. Add tracing to your code by using the `Datadog.tracer.trace` method, which you can wrap around any Ruby code:
```

File: content/en/tracing/troubleshooting/tracer_startup_logs.md [hunk 6]
```diff
@@ -8,7 +8,7 @@ further_reading:
 ---
 ## Startup logs
 
-Tracer startup logs capture all obtainable information at startup and log it either as `DATADOG TRACER CONFIGURATION` or `DATADOG TRACER DIAGNOSTICS` for easy searching within your logs.  Some languages may log to a separate file depending on language conventions and the safety of accessing `Stdout` or equivalent.  In those cases, the location of logs are noted in the language tab below.
+Tracer startup logs capture all obtainable information at startup and log it either as `DATADOG TRACER CONFIGURATION` or `DATADOG TRACER DIAGNOSTICS` to simplify searching within your logs. Some languages may log to a separate file depending on language conventions and the safety of accessing `Stdout` or equivalent.  In those cases, the location of logs are noted in the language tab below.
```
Comment (ruthnaebeck) [inline content/en/tracing/troubleshooting/tracer_startup_logs.md:11]:
```suggestion
Tracer startup logs capture all obtainable information at startup and log it either as `DATADOG TRACER CONFIGURATION` or `DATADOG TRACER DIAGNOSTICS` to simplify searching within your logs. Some languages may log to a separate file depending on language conventions and the safety of accessing `Stdout` or equivalent. In those cases, the location of logs are noted in the language tab below.
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]

**Review by ruthnaebeck — APPROVED** [review]

**Pull request merged**